adapters/tlsio_wolfssl.c

```diff
@@ -103,6 +114,18 @@ static void* tlsio_wolfssl_CloneOption(const char* name, const void* value)
                 /*return as is*/
             }
         }
+        else if(strcmp(name, OPTION_WOLFSSL_SET_DEVICE_ID) == 0 )
```

Comment:
We should be fine if these added logic sections (line 117, and line 151) that don't mention INVALID_DEVID aren't wrapped in an `#ifdef INVALID_DEVID ... #endif`.  But I will double check with my colleague what he thinks about long term maintainability and safety.
https://github.com/Azure/azure-c-shared-utility/blob/ef6ebf91a2e3530275050f0f789bab9f744c45e9/adapters/tlsio_wolfssl.c#L915-L917